### PR TITLE
Block observation get for update basic endpoint

### PR DESCRIPTION
**Pull request by sdevalapurkar**
# Overview

## Links to jira tickets

- Start of https://quartech.atlassian.net/browse/BHBC-1152

## This PR contains the following changes

- Block observation get for update basic endpoint

## This PR contains the following types of changes

- [x] New feature (change which adds functionality)
- [x] Enhancement (improvements to existing functionality)

## How Has This Been Tested?

- Needs to be properly wired up and tested/used

### CODE REVIEW

**Comment by anissa-agahchen** [comment]
/observations/1/block is broken

Screen Shot 2021-06-17 at 8 17 59 AM

File: app/src/features/projects/ProjectsRouter.tsx [hunk 1]
```diff
@@ -87,6 +87,13 @@ const ProjectsRouter: React.FC<IProjectsRouterProps> = (props) => {
         component={BlockObservationPage}
         componentProps={props}
       />
+      <PrivateRoute
+        exact
+        layout={ProjectsLayout}
+        path="/projects/:id?/surveys/:survey_id?/observations/:observation_id?/block"
```
Comment (anissa-agahchen) [inline app/src/features/projects/ProjectsRouter.tsx:93]: should this say?
path="/projects/:id?/surveys/:survey_id?/observations/:observation_id?/edit" 
or
path="/projects/:id?/surveys/:survey_id?/observations/:observation_id?/details"

Comment (sdevalapurkar) [reply app/src/features/projects/ProjectsRouter.tsx:93]: lets discuss this on call -> no this should be just block cause it is specific for block observations
I have never seen the error above so if you could show me how you reproduced it that would be awesome

Comment (anissa-agahchen) [reply app/src/features/projects/ProjectsRouter.tsx:93]: All good!   I needed to cleanout the database.

**Review by anissa-agahchen — DISMISSED** [review]
Looks great!

**Review by anissa-agahchen — APPROVED** [review]

File: app/src/interfaces/useObservationApi.interface.ts [hunk 2]
```diff
@@ -9,3 +11,25 @@ export interface IGetBlocksListResponse {
...
+
+/**
+ * Create observation post object.
+ *
+ * @export
+ * @interface ICreateObservationRequest
+ */
+export interface ICreateObservationRequest {
+  metaData: IBlockObservationForm;
+  tableData: any[][];
```
Comment (NickPhura) [inline app/src/interfaces/useObservationApi.interface.ts:23]: Heads up, there is a slight discrepancy between this definition, and the one in [Anissa's PR](https://github.com/bcgov/biohubbc/pull/367/files#diff-7978d58e06195e444e4da2d9adeda22c5478bdd772e82273c53b46f5babb5822R21).

I think we want to go with the way Anissa has it currently, where tableData is an object, with a data property that is a double array.  That way we can accommodate additional table related properties if we need to.

**Pull request merged**